### DISCUSSION TITLE
beautify/format css on save in editor

### DISCUSSION THREAD

**Discussion by scarf005** [openstyles/stylus#1692] [Ideas]
![image](https://github.com/openstyles/stylus/assets/54838975/29f5b702-195d-4114-ac1b-4f1381a9fd83)

it'd be great if editor could format on save. of course the shortcut to run beautify can be mapped to `Ctrl+S`, but this overrides `Ctrl+S` to save.

**Comment by tophf** [comment]
I somewhat doubt this should be an option because you would probably want to use it selectively. Maybe we can add a sub-button inside `Save` with its own configurable hotkey.
